bx_py_utils/test_utils/html_assertment.py

```diff
@@ -0,0 +1,92 @@
+from django.contrib.messages import get_messages
+
+
+class HtmlAssertmentMixin:
```

Comment:
Assertment is a very old-school legal term. Likely _assertion_?

Reply:
renamed